docs/resolution-getting-started.md

```diff
@@ -0,0 +1,178 @@
+# Getting Started
+
+## Introduction
```

Comment:
There's a decent amount of overlap here with getting started with pipelines in general so it seems a bit strange to have a getting started that's just based on one feature-- maybe this can be pulled out of this PR; @geriom wdyt?

Reply:
A later commit will clean up the docs. =)

Reply:
(if desired, I can port the later doc changes back into this commit, but we're getting these PRs merged quickly enough that the "final" docs content shouldn't be too many days away from being up for review)

Reply:
Actually, I'm just going to port the changes into this PR. Might as well.